Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ryanj/centos7-nodejs:6.4.0
+FROM bucharestgold/centos7-nodejs:6.9.5
```

Comment:
I was thinking that also, but i think i wanted to stay with an LTS version,  i guess i could have tagged that as :lts instead :) .  but i like version numbers